modules/alloydb/README.md

```diff
@@ -87,6 +93,23 @@ module "alloydb" {
 
 In a cross-region replication scenario (like in the previous example) this module also supports [promoting the secondary instance](https://cloud.google.com/alloydb/docs/cross-region-replication/work-with-cross-region-replication#promote-secondary-cluster) to become a primary instance via the `var.cross_region_replication.promote_secondary` flag.
 
+
+### PSC instance
+
+```hcl
+module "alloydb" {
+  source       = "./fabric/modules/alloydb"
+  project_id   = var.project_id
+  cluster_name = "db"
+  location     = var.region
+  name         = "db"
+  network_config = {
+    psc_config = {}
```

Comment:
```suggestion
    psc_config = { allowed_consumer_projects = [var.project_id] }
```
Without `allowed_consumer_projects` nobody can connect to the instance, so let's educate users how they can specify  that.